modules/configuring-your-ide.adoc

```diff
@@ -63,3 +63,60 @@ The extension you installed appears in the *Browser - Installed* list on the *Ex
 
 See link:https://open-vsx.org/[Open VSX Registry] for available third-party extensions that you can consider installing.
 
+ifndef::upstream[]
+== Building the RStudio Server notebook images
+
+To use the *RStudio Server* and *CUDA - RStudio Server* notebook images, you must first build them by creating a secret and triggering the BuildConfig, and then enable them in the {productname-short} UI by editing the `rstudio-rhel9` and `cuda-rstudio-rhel9` imagestreams.
```

Comment:
See: https://docs.openshift.com/container-platform/4.14/openshift_images/index.html

I think it should be "image streams".

Reply:
Good shout, updated.